[Oracle] Document ORA-12514 when listener service name differs from database.dbname (#7407)
debezium/debezium: documentation/modules/ROOT/pages/connectors/oracle.adoc

In the previous example, the `database.hostname` and `database.port` properties are used to define the connection to the database host.
However, in more complex Oracle deployments, or in deployments that use Transparent Network Substrate (TNS) names, you can use an alternative method in which you specify a JDBC URL.

[NOTE]
====

> In enterprise Oracle environments, the Oracle listener service name often differs from the database name specified in `database.dbname`.

Reviewer: ```suggestion
In some Oracle environments, the Oracle listener service name may differ from the database name specified in `database.dbname`.
```

> When using `database.hostname` and `database.port`, {prodname} constructs the JDBC URL using `database.dbname` as the service name.

Reviewer: ```suggestion
When using `database.hostname` and `database.port`, the {prodname} Oracle connector constructs the JDBC URL using the provided `database.dbname` as the service name.
```

> If the listener is not registered with that name, the connector fails with the following error:

Reviewer: ```suggestion
If the Oracle TNS listener is not registered with that name, the connector fails with the following error:
```

----
org.apache.kafka.connect.errors.RetriableException: Failed to resolve Oracle database version
Caused by: java.sql.SQLRecoverableException: ORA-12514: Cannot connect to database.
Service <database.dbname> is not registered with the listener at host <hostname> port <port>.
----

> In such cases, use `database.url` to specify the full JDBC connection string with the correct service name, as shown in the following example.

Reviewer: ```suggestion
This exception can be avoided by specifying the `database.url` configuration property with a full JDBC connection string that includes the service name.
An example of a connection string is shown below.
```
